python/ray/train/_internal/worker_group.py

```diff
@@ -13,7 +13,7 @@
 logger = logging.getLogger(__name__)
 
 
-class BaseWorkerMixin:
+class DistributedRayWorker:
```

Comment:
nit: Should we make this specific to Train - `DistributedTrainWorker`? Not sure if it has any benefits of making this generic.

Reply:
@richardliaw what do you think

- DistributedRayWorker
- DistributedTrainWorker
- DistributedRayTrainWorker
- DistributedWorker
- RayTrainWorker
- RayWorker
- TrainWorker

happy either way, let's make it not too long. Slight preference for `RayTrainWorker`, `DistributedRayWorker`, or `DistributedTrainWorker` (in that order)

Reply:
RayTrainWorker sounds good to me actually

Reply:
Now that you suggest it, it seems like such an obvious choice... 😃

Reply:
+1 on RayTrainWorker!